app/enterprise/2.1.x/kong-for-kubernetes/install-on-kubernetes.md

```diff
@@ -156,6 +164,14 @@ In the following steps, replace `<your-password>` with a secure password.
     ```
     $ echo '{"cookie_name":"portal_session","cookie_samesite":"off","secret":"<your-password>","cookie_secure":false,"storage":"kong"}' > portal_session_conf
     ```
+
+    If you have different subdomains for the `portal_api_url` and `portal_gui_host`, set the `cookie_domain`
```

Comment:
```suggestion
    Or, if you have different subdomains for the `portal_api_url` and `portal_gui_host`, set the `cookie_domain`
```
Putting that "or" here to hopefully stop people from doing this step twice.

Reply:
stet.

Reply:
Done